src/Protocol/Protocol_1_13.cpp

```diff
@@ -660,21 +661,25 @@ void cProtocol_1_13::HandlePacketPluginMessage(cByteBuffer & a_ByteBuffer)
 {
 	HANDLE_READ(a_ByteBuffer, ReadVarUTF8String, AString, Channel);
 
+	const auto & [Namespace, Value] = NamespaceSerializer::SplitNamespacedID(Channel);
+
 	// If the plugin channel is recognized vanilla, handle it directly:
-	if (Channel.substr(0, 15) == "minecraft:brand")
+	if (Namespace == NamespaceSerializer::Namespace::Minecraft)
 	{
-		HANDLE_READ(a_ByteBuffer, ReadVarUTF8String, AString, Brand);
-		m_Client->SetClientBrand(Brand);
+		if (Value == "brand")
+		{
+			HANDLE_READ(a_ByteBuffer, ReadVarUTF8String, AString, Brand);
 
-		// Send back our brand, including the length:
-		m_Client->SendPluginMessage("minecraft:brand", "\x08""Cuberite");
-		return;
+			m_Client->SetClientBrand(Brand);
+			m_Client->SendPluginMessage("brand", "\x08""Cuberite");  // Send back our brand, including the length.
```

Comment:
What's the advantage of putting the comment after the code? It just makes it harder to read on narrow screens.

Reply:
Nicely aligned `m_Client->S` lines... How narrow is your screen anyway xD